website/pages/docs/job-specification/hcl2/functions/ipnet/cidrsubnets.mdx

```diff
@@ -0,0 +1,94 @@
+---
+layout: docs
+page_title: cidrsubnets - Functions - Configuration Language
```

Comment:
```suggestion
page_title: cidrsubnets - Functions - Configuration Language
sidebar_title: cidrsubnet
```

So that the sidebar isn't `cidrsubnets - Functions - Configuration Language`

Reply:
Oops - I missed this comment when I merged :(